sopel/cli/plugins.py

```diff
@@ -319,8 +319,12 @@ def handle_configure(options):
         return 0  # nothing to configure is not exactly an error case
 
     print('Configure %s' % plugin.get_label())
-    plugin.configure(settings)
-    settings.save()
+    try:
+        plugin.configure(settings)
+        settings.save()
```

Comment:
My instinct says this should be outside the `try` so as to avoid saying that the file wasn't modified even if `KeyboardInterrupt` occurred after `settings.save()` was called (and therefore the file might have been modified after all).

Reply:
Yes. Will do.